src/panels/config/areas/ha-config-area-page.ts

```diff
@@ -131,6 +135,16 @@ class HaConfigAreaPage extends LitElement {
       this.entities
     );
 
+    const sceneEntities = entities.filter(
```

Comment:
We have a groupBy helper that could help do this in 1 pass `grouped = groupby(entities, ent => computeDomain(entity.entity_id))`